packages/appium/docs/en/guides/caps.md

```diff
@@ -67,6 +67,15 @@ Each Appium client has its own way of constructing capabilities and starting a s
 examples of doing this in each client library, head to the [Ecosystem](../ecosystem/index.md) page
 and click through to the appropriate client documentation.
 
+## BiDi
+
+Appium can support [WebDriver BiDi](https://w3c.github.io/webdriver-bidi/) protocol.
```

Comment:
can support -> supports

Reply:
it would be useful to mention since which version the protocol is supported. It is also important to mention that not all commands are supported that the standard defines.

Reply:
Maybe base–driver version then? This doc will be for Appium 2.12 or newer in appium.io